public/index.html

```diff
@@ -14,6 +14,18 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>QPP Submissions API Developer Documentation</title>
+    <!--START Google Analytics tracking code - for qpp.cms.gov-->
+    <script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-15356370-63', 'auto');
```

Comment:
looks like we need to edit the tracking code since this is a secondary domain:
```js
ga('create', 'UA-15356370-63', 'auto', {'allowLinker': true});
ga('require', 'linker');
ga('linker:autoLink', ['qpp.cms.gov'] );
ga('send', 'pageview'); // <-- this was already here
```
from step 3 on https://support.google.com/analytics/answer/1034342?hl=en